mautic/mautic
# add do not contact field for matching

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | y
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
In CRM integrations we want the ability to map do not contact state to integration fields.

#### Steps to test this PR:
1. Apply this PR, you should see a new contact field called do not contact
2. this will return a value of 0 or 1 (1 is contactable 0 is not contactable)
